core/shared/src/main/scala/cats/parse/Parser.scala

```diff
@@ -1311,6 +1345,15 @@ object Parser extends ParserInstances {
           pa
       }
 
+    def expect1[A](p: Parser[A]): Parser1[A] =
```

Comment:
Probably worth adding a scaladoc comment on what this method does and how we're using it.

Reply:
Note this in the `private object Impl` so we could document it but I don't think it will show up in any scala doc since it isn't part of the public API.

What do you think? My view was that given the above we didn't need to document it (since it is also so short anyone seeing the code can see what it is doing, or guess from the type signature and name).

Reply:
Yeah that's fair. I missed that it was defined in a private scope.